source/stylesheets/site.css.scss

```diff
@@ -1,10 +1,18 @@
 //this is where we import all our sass partials
 
 //vendor setup
-@import 'normalize';
 
+@import 'normalize';
+@import 'bourbon';
+@import 'fonts';
 @import 'variables';
 @import 'typography';
 @import 'base';
 @import 'card';
+@import 'card-back.scss';
 @import 'header';
+
+details::-webkit-details-marker {
```

Comment:
We have this `-webkit-details-marker` declaration elsewhere, so it can be deleted